src/libecalc/presentation/yaml/file_context.py

```diff
@@ -0,0 +1,50 @@
+from dataclasses import dataclass
+from typing import Optional
+
+from libecalc.presentation.yaml.yaml_entities import YamlDict
+
+
+@dataclass
+class FileMark:
+    line_number: int
+    column_number: int
+
+    def __str__(self) -> str:
+        return f" line {self.line_number}, column {self.column_number}"
+
+
+@dataclass
+class FileContext:
```

Comment:
can this also be used for other resources than yaml, such as csv? Relevant?

Reply:
Sure, that would make sense to me. I'm hoping we can improve on this when looking into validation (of csv resources)